src/DataProtection/DataProtection/src/KeyManagement/KeyManagementOptions.cs

```diff
@@ -94,6 +94,25 @@ internal static TimeSpan MaxServerClockSkew
         }
     }
 
+    /// <summary>
+    /// During each default key resolution, if a key decryption attempt fails,
+    /// it can be retried, as long as the total number of retries across all keys
+    /// does not exceed this value.
+    /// </summary>
+    /// <remarks>
+    /// Settable for testing.
+    /// </remarks>
+    internal int MaximumDefaultKeyResolverRetries { get; set; } = 10;
```

Comment:
Does it make sense to have `Global` in this name to indicate that this is a cumulative count across all keys?

Reply:
I'm not sure about "Global" specifically, but it needs something.  I'll think about it.

Reply:
"Cummulative" might be another alternative.

Reply:
I went with "total" to match the doc comment.